src/routes/console/wizard/cloudOrganization/organizationDetails.svelte

```diff
@@ -29,9 +29,9 @@
         $createOrgSteps = updateStepStatus($createOrgSteps, 4, false);
     }
 
-    $: freePlan = $plansInfo.plans.find((p) => p.$id === 'tier-0');
-    $: proPlan = $plansInfo.plans.find((p) => p.$id === 'tier-1');
-    $: scalePlan = $plansInfo.plans.find((p) => p.$id === 'tier-2');
+    $: freePlan = $plansInfo.get('tier-0');
+    $: proPlan = $plansInfo.get('tier-1');
+    $: scalePlan = $plansInfo.get('tier-2');
```

Comment:
Don't forget to change the magic strings to constants after merging the other PR!